Enabling Swift support is normally intended for building an overlay for a
Swift SDK, which changes the installation layout. Prevent this.

(The architecture subdirectory that used to be patched out here is now
controlled by the upstream `dispatch_INSTALL_ARCH_SUBDIR` option, which
defaults to NO.)

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -319,7 +319,7 @@ configure_file("${PROJECT_SOURCE_DIR}/cmake/config.h.in"
 add_compile_definitions($<$<COMPILE_LANGUAGE:C,CXX>:HAVE_CONFIG_H>)


-if(ENABLE_SWIFT)
+if(0)
   include(SwiftSupport)
   option(dispatch_INSTALL_ARCH_SUBDIR "Install libraries under an architecture subdirectory" NO)

--- a/man/CMakeLists.txt
+++ b/man/CMakeLists.txt
@@ -1,6 +1,6 @@

 # TODO(compnerd) add symlinks
-if(NOT ENABLE_SWIFT)
+if(1)
   install(FILES
             dispatch.3
             dispatch_after.3
